coderd/database/migrations/000223_notifications_metrics.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE notification_messages
+ADD COLUMN queued_seconds INTEGER NULL;
```

Comment:
Do we really to store this on the row?  It feels a little strange and ambiguous.  If the notification_message is currently queued, then it's a bit meaningless, since we don't know for how much longer it will be queued. I.e. it is at present being returned on Enqueue, but is meaningless in that context.

When we dequeue it, can't the notifier compute how long it was on the queue based on it's `created_at` or `updated_at` time?  Or, if we really want to ensure all the measurements are based on the central PostgreSQL server clock, then it can part of the result returned to the notifier without having to write it back to the database.